tests/e2e/vsphere_volume_expansion.go

```diff
@@ -736,7 +736,7 @@ var _ = ginkgo.Describe("Volume Expansion Test", func() {
 			if guestCluster {
 				expectedErrMsg = "didn't find a plugin capable of expanding the volume"
 			} else {
-				expectedErrMsg = "failed to expand volume"
+				expectedErrMsg = "failed to retrieve snapshots for volume"
```

Comment:
This is not correct...instead just check if error occurred and log the error

Reply:
+1